exercicios/para-sala/.gitignore

```
@@ -1 +1,2 @@
 *node_modules/
+
```

oq acha de ignorarmos o env tbm?
```suggestion
*.env
```